### PR TITLE
V1.3 - UG 

### PR DESCRIPTION
**Pull request by zhing22**
suggestions: 
* right now `add group` `add assessment` names search is case sensitive, can make it non case-sensitive since it dosent clash with ending up with multiple matches
* make content page not auto generated
* having the back to table of content everywhere is a bit...

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -172,36 +188,56 @@ Examples:
...
+* Using `<index>`, `-n <student_name>` or `-i <student_id>` displays the information of the particular student's performance in all his graded assessments. 
+* The `<index>` refers to the index number shown in the displayed student list. The `<index>` must be a positive integer 1, 2, 3, …
+* Using `-g <group_name>` displays the information of the group's performance in all their graded assessments. 
+* Using `-a <assessment_name>` displays the information of the cohort's performance in the particular assessment.
+* Does not support partial searching. e.g. `T01` does not match `T01A`. Full name must be entered. 
+* Entering `-f <export_file_path>` exports the graph produced from the command to the specified location. 
+
+Examples:
+* `show -n Alex Yeoh` displays line chart of Alex Yeoh's performance in his assessments. 
+* `show 2 -f` exports the line chart produced of the 2nd student in the address book.
```
Comment (ganhongyao) [inline docs/UserGuide.md]:
```suggestion
* `show 2 -f chartImage` shows and exports the line chart produced of the 2nd student in the student list to a file `chartImage.png`. 
```

**Review by ganhongyao — COMMENTED** [review]
Thanks a lot for helping to update the UG! 

I think there are still some references to "address book", could we replace them with "Source Control" or "student list"?

**Review by jonas-chow — APPROVED** [review]
lgtm, thanks for doing this!

**Comment by zhing22** [comment]
i haven't finish though haha, probably gonna take me some time but ill try to finish soon so everyone can check

**Comment by jonas-chow** [comment]
> make content page no auto generated

What's wrong with that though? I feel like how the content page works right now works just fine. What specifically do you not like about it?

> having the back to table of content everywhere is a bit...

You're free to remove the return to table of contents if you find that it's a little too much...

Personally I think it's basically required to have a function like this for ease of access. I put it in every section so that regardless of what section you click on from the table of contents, you are able to return to the table of contents at any time. 

I think the only other way that this would work is if you have a return to top button that stays on somewhere on the page. The only problem is I have no idea how to do this with pure markdown.

**Comment by jonas-chow** [comment]
A few suggestions:

Should we make all command words in **`bold code format`** like those listed in the Quick Start section?

Also, I think *CSV* could be added into glossary: A delimited text file that uses a comma to separate values, generally used to move data between programs such as Microsoft Excel and Google Sheets.

Perhaps we could also an image in the import section for the csv format.

I'll make these changes on my own!

**Comment by zhing22** [comment]
is this safe to merge directly? or will the other file changes mess things up :(

**Comment by jonas-chow** [comment]
It's safe. I'll merge it now.

**Pull request merged**